between random: 7458106
between random: 7458106
between random: 7458098
between random: 7458106
between random: 7458106
between random: 36610
between random: [253,252,126,10,248,130,69,176]
between random: [253,252,126,10,248,130,69,176]
between random: [237,236,242,116,42,168,143,212]
between random: [237,236,242,116,42,168,143,212]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [248,47,251,2,2,33,48,236]
between random: [90,0,63,217,61,195,183,24]
between random: [90,0,63,217,61,195,183,24]
"    require ~sent_pkt(scid,pkt.ptype,pkt.seq_num);  # [4]
"
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_packet.ivy: line 397: error: assumption failed
output directory: temp/206
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/quic-go/server/
implementation command: ./server -p 4443 127.0.0.1
../quic_server_test_newcoid_rtp_error (0) ...
implementation command: ['./server', '-p', '4443', '127.0.0.1']
quic_process pid: 6129
timeout 100 ./build/quic_server_test_newcoid_rtp_error seed=339 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
